bibliopixel/drivers/visualizer.py

```diff
@@ -1,11 +1,6 @@
-import time
-
-import os
-import platform
-import subprocess
-from network import DriverNetwork, socket
-import math
+import time, os, platform, subprocess, math
```

Comment:
You know what... screw it. I'll find whatever stupid PEP rule that is an exclude it in my flake8 config. I'm just used to it because that's what's mandated at RedHat.